src/Core/src/VisualDiagnostics/VisualDiagnosticsOverlay.Android.cs

```diff
@@ -64,10 +64,16 @@ Point GenerateAdornerOffset(View graphicsView)
 			if (nativeActivity.Resources == null || nativeActivity.Resources.DisplayMetrics == null)
 				return new Point();
 
-			float dpi = nativeActivity.Resources.DisplayMetrics.Density;
-			float heightPixels = nativeActivity.Resources.DisplayMetrics.HeightPixels;
+			var decorView = nativeActivity.Window?.DecorView;
+			var rectangle = new Android.Graphics.Rect();
 
-			return new Point(0, -(heightPixels - graphicsView.MeasuredHeight) / dpi);
+			if (decorView is not null)
```

Comment:
What happens if this is null? Previously there was some logic that now will just be 0. Is that correct?

Reply:
If it's null, then it uses the default `rect` values, so 0. `0 / dpi` should be 0, which is what I would expect.